tarynmartin/shopify-challenge
# Feature/nominate functionality

#### What’s this PR do?
- Add onClick function to allow a user to nominate a movie
- Add onClick function to allow a user to remove a nomination
- Add ability to disable nominate button so user can't nominate a movie twice

#### Where should the reviewer start?
- In App.js to see helper functions
- Results & Nominate.js to see how helper functions are passed down
- MovieCard.js to see logic for onClick functionality & disabling buttons

#### How should this be manually tested?
- in terminal, run npm start
- in browser, search for a movie
- nominate all movies; buttons for selected movies should be disabled
- click remove on nominated movies to see them removed

#### What are the relevant tickets?
resolves #5 resolves #7
